tools/src/code-review/index.ts

```diff
@@ -9,10 +9,25 @@ import reviewChangelogEntries from './reviewers/reviewChangelogEntries';
 import reviewForbiddenFiles from './reviewers/reviewForbiddenFiles';
 import { ReviewEvent, ReviewComment, ReviewInput, ReviewOutput, ReviewStatus } from './types';
 
+const DISABLE_CHANGELOG_MAGIC_COMMENT = '<!-- disable:changelog-checks -->';
+
 /**
  * An array with functions whose purpose is to check and review the diff.
  */
-const REVIEWERS = [checkMissingChangelogs, reviewChangelogEntries, reviewForbiddenFiles];
+const REVIEWERS = [
+  {
+    action: checkMissingChangelogs,
+    magicCommentToDisable: DISABLE_CHANGELOG_MAGIC_COMMENT,
```

Comment:
Maybe it would be better to give them an ID, so that you can use `<!-- disable:<id> -->` for any review

Reply:
In this case - should all of them have IDs and be "disablable" (able to be disabled 😜 )?  like I did now